app.vue

```diff
@@ -70,8 +71,10 @@ const routerBackProfileLoad = async () => {
       // only makes sense to load profile if it's not already loaded
       if (!storeProfile) {
         try {
+          isLoadingProfile.value = true
           await fetchAndStoreProfile(toProfileAddress)
           fetchAndStoreAssets(toProfileAddress)
```

Comment:
There needs to be an await here I think

Reply:
I removed await so we can show profile and not wait for loading it's assets